### ISSUE TITLE
can i ask a question?

### ISSUE THREAD

**Issue by tuoniaoren** [LynnHo/DCGAN-LSGAN-WGAN-WGAN-GP-Tensorflow#4]
i use the code whitch be used  to train cartoon pictures with WGAN-GP. i don't know what the mean of   **n_critic = 5** , and why do you to set it. thanks.

**Comment by LynnHo** [comment]
@tuoniaoren This is a hyperparameter following the WGAN-GP paper, which controls the number of D steps in each iteration.

**Comment by tuoniaoren** [comment]
thanks，last week,i use DCGAN to generate my dataset ,but its loss is very bad , the pictures that it generate are also bad. today i use WGAN-GP ,the picture is better than DCGAN.

**Comment by LynnHo** [comment]
@tuoniaoren Yes, DCGAN is not stable and hard to be tuned, WGAN-GP is a very stable method with better quality. There are many gan techniques for stable training and good quality, here is my implementation of some useful techniques https://github.com/LynnHo/GAN-Techniques-Tensorflow.
